content/_index.md

```diff
@@ -7,10 +7,26 @@ BookToC: false
 
 # The Linux Userspace API (UAPI) Group
 
-The userspace API ("uapi") group is a community for people with an interest in innovating how we build, deploy, and run modern Linux operating systems.
+The userspace API ("uapi") group is a cross-distro community for people with an interest in innovating how we build, deploy, run, and safely update modern Linux operating systems.
```

Comment:
Maybe we can list the communities and distributions that are publicly endorsing the effort?

Reply:
I would phrase it as "contributing members" instead of making claims about "endorsement", but generally yes. A good idea.

Reply:
Added an incomplete initial list of contributors. Org members are invited to file PRs to update / extend the list.